src/core/rendering.js

```diff
@@ -119,6 +119,20 @@ p5.prototype.createCanvas = function(w, h, renderer) {
   }
   this._renderer.resize(w, h);
   this._renderer._applyDefaults();
+
+  //isWebGL flag is set and made read-only
+  if (this.isWebGL == false) {
+    Object.defineProperty(this,"isWebGL" , {
+      value: this._renderer.drawingContext instanceof WebGLRenderingContext,
```

Comment:
If we use a `get` accessor here instead of `value` directly, then we can put this in the constructor for p5 and not have to worry about handling `createCanvas` not being called or being called more than once, since the value will be determined at runtime. i.e., `Object.defineProperty(this, 'webgl', { get: () => this._renderer.drawingContext instanceof WebGLRenderingContext, writable: false })`.

Reply:
Can we also add the same accessor to the `p5.Graphics` constructor?